{
  "_annotation": {
    "cve_id": "CVE-2024-25128",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-25128.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:flask-appbuilder_project:flask-appbuilder:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "FF52CE43-91CD-5A69-A0DB-656B3D3C92F5",
                "versionEndExcluding": "4.3.11",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}